### ISSUE TITLE
Closing and opening a window with hides the listview

### ISSUE THREAD

**Issue by jonasfunk** [iskugor/Ti.SwipeRefreshLayout#22]
I am developing an Android app, where I am forced to run it as singleTask. This means that when returning to the app, either from clicking the app icon or from another app, all activities but the root are closed. The root window is closed as well, but just needs to be opened again by calling the open() method. This works fine, but it seems like the listview hooked up to the Ti.SwipeRefreshLayout is not redrawn/not visible after that. As a test, I tried to use the listview without Ti.SwipeRefreshLayout, and then it's visible after reopening the app.

**Comment by iotashan** [comment]
I have the exact same issue. Point of fact... it's not the child that is not visible, it's the actual SwipeRefreshLayout that doesn't redraw. I'm not seeing any obvious errors in the console output.

**Comment by annakozy2011** [comment]
Same here, but it happens when I set the parent view invisible, then back visible. 
I am also forced to run the app as Single Task, but I don't think it is related. I have  simple tabgroup simulation, so I am switching the views on and off, and eventually the listview hooked to refresh control disappears...

**Comment by annakozy2011** [comment]
For my issue I found the reason here:

https://github.com/appcelerator/titanium_mobile/pull/6767

I use ScrollableView instead of tabgroup and it should be given correct cacheSize (the property is already implemented in Appcelerator)

**Comment by DFoxinator** [comment]
@iotashan @jonasfunk did either of you ever figure this out?

**Comment by iotashan** [comment]
no

**Comment by annakozy2011** [comment]
Do you have anything like a scrollable view in your window instead of tabgroup? For me it was not the issue with the module, but the issue that the scrollable view was not caching correctly. I was going to jump out from the window, when I found "cacheSize" property and increased it, and that was it.

**Comment by jonasfunk** [comment]
Not experiencing the issue anymore, but unfortunately I cannot remember what I did to fix it. Looking at the code, I can see that I am using [Fokkes pullToRefresh widget](https://github.com/FokkeZB/nl.fokkezb.pullToRefresh). Are you using that one?

**Comment by annakozy2011** [comment]
Yes, so did I, when I subscribed to this issue, but for me it was because of cacheSize. It does not do much,  the widget, it´s just a shortcut to have the same code for iOS and Android.

**Comment by annakozy2011** [comment]
It definitely disappears when you kinda try to take it out from memory then construct it again (like when using scrollable view, and I guess also when closing/reopening the window without constructing it from zero). The workaround is just "dont do it"

**Comment by DFoxinator** [comment]
Hmm, thanks for the information. I am not using any scrollable views. I am using this module directly, and not Fokkes pullToRefresh widget.

**Comment by jonasfunk** [comment]
I think I tried using the module directly, so even though it doesn't makes sense I'm guessing the fix has something to do with the widget, since we don't use the pullToRefresh widget on iOS. You can give it a try.

**Comment by Lenndev** [comment]
Is there anyone with a concrete answer? For some reason i have a problem like this, but with a tableview instead of a listview. on android. I've used the fokkeZandbergen plugin before, then switched to using this plugin directly, but no changes. Initially it works. But when i open a window it is not displayed
